pymatbridge/publish.py

```diff
@@ -1,5 +1,10 @@
-import IPython.nbformat.v4 as nbformat
-from IPython.nbformat import write as nbwrite
+try:
+    import nbformat.v4 as nbformat
+    from nbformat import write as nbwrite
+except:
```

Comment:
Thank you for your continued patience, which is much more like tutoring. 

Would have been done faster. I ran into a problem where the clone on my machine was causing a hang in the desktop app. It took me a long time to figure out that I hard to delete the clone to get the app to unfreeze. (Not I need to submit another bug report- apps shouldn't hang like that!). 


> On Aug 29, 2015, at 10:37 AM, Steven Silvester <notifications@github.com> wrote:
> 
> In pymatbridge/publish.py <https://github.com/arokem/python-matlab-bridge/pull/213#discussion_r38262596>:
> 
> > @@ -1,5 +1,10 @@
> > -import IPython.nbformat.v4 as nbformat
> > -from IPython.nbformat import write as nbwrite
> > +try:
> > +    import nbformat.v4 as nbformat
> > +    from nbformat import write as nbwrite
> > +except:
> Please explicitly use except ImportError, to avoid hiding any other kind of error.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/arokem/python-matlab-bridge/pull/213/files#r38262596>.
>